CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to
 - cosmwasm-std: Added new `EurekaMsg` and `CosmosMsg::Eureka` variant ([#2340])
 - cosmwasm-std: Implement downcasting for `Api` trait. This allows using
   `MockApi::addr_make` from `DepsMut`. ([#2383])
+- cosmwasm-std: Added `EurekaPort` to `ContractInfoResponse`. ([#2390])
```

Comment:
We usually link the PR. Also the definition of that link is missing. See below, e.g.
`[#2155]: https://github.com/CosmWasm/cosmwasm/issues/2155`

Reply:
Right. Thanks for pointing that out